### PR TITLE
add TensorSpace.js to the gallery

### PR DESCRIPTION
**Pull request by syt123450**
#### Description

Add TensorSpace.js to gallery.

---

##### For repository owners only:

Please remember to apply all applicable tags to your pull request.
Tags: FEATURE, BREAKING, BUG, PERF, DEV, DOC, SECURITY

For more info see: https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/909)

### CODE REVIEW

**Comment by syt123450** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).

I signed it!

File: GALLERY.md [hunk 1]
```diff
@@ -80,3 +80,4 @@ These tutorials augment the [official tutorials](https://js.tensorflow.org/tutor
 - [Handsfree.js](https://github.com/handsfreejs/handsfree) - A library for adding face-controlled pointers to your site.
 - [face-api.js](https://github.com/justadudewhohacks/face-api.js) - A JavaScript API for face detection and face recognition in the browser.
 - [TensorFlow.js GPU-accelerated t-SNE](https://github.com/tensorflow/tfjs-tsne) - A linear t-SNE for the web.
+- [TensorSpace.js](https://github.com/tensorspace-team/tensorspace) - Neural network 3D visualization framework, build interactive and intuitive model in browsers, support pre-trained deep learning models from TensorFlow, Keras, TensorFlow.js.
```
Comment (caisq) [inline GALLERY.md]: Please fix punctuation and grammar. I suggest: 

A 3D visualization framework for neural networks, featuring interactive and intuitive display of models in the browser. Supports pretrained models from TensorFlow, Keras, and TensorFlow.js.

Comment (syt123450) [reply GALLERY.md]: Thanks for your review @caisq ! Punctuation and grammar fixed.

Comment (caisq) [reply GALLERY.md]: np, @syt123450  Thanks for adding TensorSpace to the gallery. Very cool project!

**Pull request merged**